.github/workflows/hopeit-aws-test-pypi-publishing.yml

```diff
@@ -32,4 +32,4 @@ jobs:
       env: 
         TEST_PYPI_API_TOKEN: ${{ secrets.TEST_PYPI_API_TOKEN_ALL }}
       run: |
-        make PLUGINFOLDER=plugins/aws/s3 publish-plugin-pypi-test
+        make PLUGINFOLDER=plugins/aws/s3/ publish-plugin-pypi-test
```

Comment:
why not to remove the final dash here and put it when $(PLUGINFOLDER) variable is used, like in hopeit.engine?

Reply:
This change prevents unintended behavior in the previous code:

```
clean-dist-plugin:
	rm -rf $(PLUGINFOLDER)/dist
```
Without it, calling `make clean-dist-plugin` results in `rm -rf /dist`.

Now it is:

```
clean-dist-plugin:
	rm -rf $(PLUGINFOLDER)dist
```
I think we should apply the same change in `hopeit.engine` to prevent unexpected behaviors.